Makefile

```diff
@@ -42,3 +42,52 @@ build:
 .PHONY: clean
 clean:
 	rm -rf integration/testdata/fixtures/
+
+$(GOBIN)/bbolt:
+	go get -v github.com/etcd-io/bbolt/cmd/bbolt
+
+DB_TYPE ?= trivy
+
+ifeq ($(DB_TYPE),trivy-light)
+	DB_ARG := --light
+else
+	undefine DB_ARG
+endif
+
+trivy-db:
+	make build
+
+.PHONY: db-all
+db-all:
+	make build db-fetch
+	make db-build
+	make db-compact
+	make db-compress
+
+.PHONY: db-fetch
+db-fetch:
+	mkdir -p cache/vuln-list cache/ruby-advisory-db cache/rust-advisory-db cache/php-security-advisories cache/nodejs-security-wg cache/python-safety-db
+	wget -qO - https://github.com/aquasecurity/vuln-list/archive/master.tar.gz | tar xz -C cache/vuln-list --strip-components=1
+	wget -qO - https://github.com/rubysec/ruby-advisory-db/archive/master.tar.gz | tar xz -C cache/ruby-advisory-db --strip-components=1
+	wget -qO - https://github.com/RustSec/advisory-db/archive/master.tar.gz | tar xz -C cache/rust-advisory-db --strip-components=1
+	wget -qO - https://github.com/FriendsOfPHP/security-advisories/archive/master.tar.gz | tar xz -C cache/php-security-advisories --strip-components=1
+	wget -qO - https://github.com/nodejs/security-wg/archive/master.tar.gz | tar xz -C cache/nodejs-security-wg --strip-components=1
+	wget -qO - https://github.com/pyupio/safety-db/archive/master.tar.gz | tar xz -C cache/python-safety-db --strip-components=1
+
+.PHONY: db-build
+db-build: trivy-db
```

Comment:
Is there any reason you don't specify `build` directly?

```suggestion
db-build: build
```

Reply:
If I'd depend on the phony target build here, trivy would get rebuilt every time you call db-build

Reply:
I understand.